staging/src/k8s.io/apiserver/plugin/pkg/audit/buffered/buffered_test.go

```diff
@@ -173,7 +173,7 @@ func TestBufferedBackendShutdownWaitsForDelegatedCalls(t *testing.T) {
 
 	// Run backend, process events, wait for them to be batched and for delegated call to start.
 	stopCh := make(chan struct{})
-	backend.Run(stopCh)
+	go backend.Run(stopCh)
```

Comment:
`run` is a blocking method now. It waits for all listeners it started to finish. So i think we can use the goroutine for this.